doc/rados/operations/placement-groups.rst

```diff
@@ -100,8 +100,8 @@ The output will resemble the following::
   data of other pools). 
 
 - **RATIO** is the ratio of (1) the storage consumed by the pool to (2) the
-  total raw storage capacity. In order words, RATIO is defined as (SIZE * RATE)
-  / RAW CAPACITY.
+  total raw storage capacity. In order words, RATIO is defined as 
+  (SIZE * RATE) / RAW CAPACITY.
```

Comment:
I have left this expression unaltered. This is what mathematical expressions look like when they have not been rendered by the :math: extension. Several mathematical expressions are left in this form here, so that readers can compare them to the expressions that have been processed by the :math: extension.